notebooks/102-pytorch-onnx-to-openvino/102-pytorch-onnx-to-openvino.ipynb

```diff
@@ -104,14 +115,19 @@
    },
    "outputs": [],
    "source": [
-    "print(\"Downloading the Fastseg model (if it has not been downloaded before)....\")\n",
-    "model = MobileV3Large.from_pretrained().cpu().eval()\n",
-    "print(\"Loaded PyTorch Fastseg model\")\n",
-    "\n",
-    "# Save the model.\n",
-    "model_path.parent.mkdir(exist_ok=True)\n",
-    "torch.save(model.state_dict(), str(model_path))\n",
-    "print(f\"Model saved at {model_path}\")"
+    "print(\"Downloading the DeepLabV3 Mobilenet v3 Large model (if it has not been downloaded before)....\") \n",
+    "download_file(DeepLabV3_MobileNet_V3_Large_Weights.COCO_WITH_VOC_LABELS_V1.url, filename=weights_path.name, directory=weights_path.parent)\n",
+    "# create model object\n",
+    "# our pretrained model was trained with using aux_loss,\n",
+    "# we need specify this parameter for avoiding keys mismatch in weights state dict.\n",
+    "model = deeplabv3_mobilenet_v3_large(aux_loss=True)\n",
+    "# read state dict, use map_location argument for avoiding situation when weights saved on cuda which can be unavailable in system\n",
```

Comment:
```suggestion
    "# read state dict, use map_location argument to avoid a situation where weights are saved in cuda (which may not be unavailable on the system)\n",
```